docs/Rakefile

```diff
@@ -86,7 +86,13 @@ task :adoc_tar => [:init, :adoc_variables, :markdown_to_asciidoc] do
         tar.add_file(f, 0444) { |io|
           if File.extname(f) == ".adoc" then
             adoc_content = File.read(f)
-            adoc_content = adoc_content.gsub(/include::.*variables\.adoc\[\]/, "include::minishift/variables.adoc[]")
+            adoc_content.gsub!(/include::.*variables\.adoc\[\]/, "include::minishift/variables.adoc[]")
+            # skip writing-docs.adoc file since it contains match regex pattern
```

Comment:
what is this about? No better way to fix that. Having to exclude writing-docs explicitly sucks.